demo/src/examples/1 User Interaction.svelte

```diff
@@ -3,19 +3,32 @@
   import { examples } from "../router";
   import { location } from "svelte-spa-router";
   import AppSidebar from "../components/AppSidebar.svelte";
-  import maplibregl from "maplibre-gl";
+  import maplibregl, { Map } from "maplibre-gl";
```

Comment:
I specifically use exclusively namespaced maplibre's features. Even for types. Like `const map: maplibregl.Map`, but not `const map: Map`. Please, check the usages and change not to import namespaced features.

Reply:
changed in this file and updated the already existing third example page to reflect the same methodology